src/CADL.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -133,7 +133,9 @@ function getCSharpInputTypeKindByIntrinsicModelName(
             return InputTypeKind.Boolean;
         case "plainDate":
             return InputTypeKind.Date;
-        case "zonedDateTime":
+        case "utcDateTime":
+            return InputTypeKind.DateTime;
+        case "offsetDateTime":
```

Comment:
Just curious what change needed for the serialization and deserialization for those new datetime types.
As this should not be a breaking change, we could decide that later.

Reply:
Discussed with Mike,  no change needed in codegen for serialization and deserialization